### PR TITLE
Awesome Books ES6: Refactor awesome books with es6

### PR DESCRIPTION
**Pull request by AmanWorku**
# Activities done

- Used modules to refactor the Awesome Books page.
- Kept all functionalities of the app without errors.
- Implement ES6 syntax to the functionalities in JavaScript.
- Organized modules in an appropriate way.
- Used Luxon library to display date and time.

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @AmanWorku,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly ensure your linters are set up and running. Also ensure you have a descriptive readme using the template provided by Microverse

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by FrederickMih** [comment]
## Hi :raising_hand:  @AmanWorku , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: 

**Linters checks are not still running on this PR.**
Screenshot from 2022-12-05 21-22-01
**Kindly make sure that GitHub action is enabled.**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalidated* :red_circle: 

Hi @AmanWorku, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] does not follow Microverse [updated template](https://github.com/microverseinc/readme-template) for README file. Kindly follow [this template](https://github.com/microverseinc/readme-template) to make your README and feel free to customise it to suite your project needs.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (Rustamxon7) [inline .eslintrc.json:1]: **You did a great job. I love the styling of your project.** 

[OPTIONAL] Kindly add MIT license to your project :)

**Review by Rustamxon7 — APPROVED** [review]
Hi @AmanWorku, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. You did a great job. I love the styling of your project 💯

## Optional suggestions
[OPTIONAL] Kindly add MIT license to your project :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**